supabase/migrations/20260725112951_register_billing_period_stats_email_cron.sql

```
@@ -0,0 +1,182 @@
+    cycle_end := date_trunc('month', p_as_of)
+      + ((v_this_anniv_dom - 1) || ' days')::interval;
+  ELSE
+    cycle_start := NULL;
+    cycle_end := NULL;
+  END IF;
+
+  RETURN NEXT;
+END;
+$$;
+
+ALTER FUNCTION public.billing_period_completed_cycle(timestamptz, date) OWNER TO postgres;
```

SQLFluff (4.2.2)

[error] Line is too long (90 > 80).

(LT05)